compiler/modules/print.llvm/llvm.bal

```diff
@@ -117,27 +117,29 @@ public class Module {
 
     // Corresponds to LLVMAddFunction
     public function addFunctionDefn(string name, FunctionType fnType) returns FunctionDefn {
-        if name is IntrinsicFunctionName {
+        string fnName = escapeIdent(name);
```

Comment:
You've got this common code in three places (escape and panic if it's an intrinsic function name), so I suggest you factor this out into a separate function (e.g. `escapeGlobalIdent`).

Reply:
Fixed